packages/twenty-front/src/modules/object-record/record-table/components/RecordTable.tsx

```diff
@@ -207,6 +209,14 @@ export const RecordTable = ({
 
   const visibleTableColumns = useRecoilValue(visibleTableColumnsSelector());
 
+  const { tableRowIdsState } = useRecordTableStates(recordTableId);
+  const tableRowIds = useRecoilValue(tableRowIdsState);
+
+  const { loading } = useLoadRecordIndexTable(
```

Comment:
RecordIndex is the parent page which loading either the RecordTable or the RecordBoard. We should not call a recordIndex hook within the table, the dependency is in the wrong way